app/helpers/markdown_helper.rb

```diff
@@ -0,0 +1,12 @@
+module MarkdownHelper
+  def markdown(text)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML,
+                                       no_intra_emphasis: true,
+                                       fenced_code_blocks: true,
+                                       autolink: true,
+                                       tables: true,
+                                       underline: true,
+                                       highlight: true)
+    markdown.render(text).html_safe
```

Comment:
Should there be a test in the helper method before rendering or a test of the helper or a separate spec for the renderer?